crates/core/tedge_agent/src/tedge_to_te_converter/converter.rs

```diff
@@ -78,6 +78,7 @@ impl TedgetoTeConverter {
         alarm.insert("severity".to_string(), severity.into());
         message.topic = te_topic;
         message.payload = serde_json::to_string(&alarm)?.into();
+       let message  = message.with_retain();
```

Comment:
To be consistent with the previous lines:

```suggestion
       message.retain = true;
```